src/components/resources/index.js

```diff
@@ -129,13 +129,13 @@ const RESOURCES = [
         title: "Wallet SDK Swift",
         Svg: Graphics.walletSdkSwift,
         content: "Build Swift-based applications using our SDKs",
-        linkTo: "https://input-output-hk.github.io/atala-prism-wallet-sdk-swift/"
+        linkTo: "https://input-output-hk.github.io/atala-prism-wallet-sdk-swift/documentation/atalaprismsdk/"
```

Comment:
Could you explain why changing the path? The swift SDK is working well in the current staging website. It was changed to https://input-output-hk.github.io/atala-prism-wallet-sdk-swift/documentation/edgeagentsdk/ in the PR https://github.com/input-output-hk/atala-prism-docs/pull/93

Reply:
reverted back to original link, we will do a link check in second iteration.

Reply:
https://input-output-hk.github.io/atala-prism-wallet-sdk-swift/
does not work

https://input-output-hk.github.io/atala-prism-wallet-sdk-swift/documentation/edgeagentsdk/ 
Looks fine :)